### PR TITLE
Added 2 more test examples to ModelicaTest.Electrical.Machines

### PR DESCRIPTION
**Pull request by AHaumer**
These examples test the proper handling of parameter records.

### CODE REVIEW

File: ModelicaTest/Electrical/Machines.mo [hunk 1]
```diff
@@ -207,4 +207,382 @@ Default machine parameters of model <a href=\"modelica://Modelica.Electrical.Mac
...
+        annotation (Line(points={{60,0},{80,0},{80,-20}}, color={0,0,0}));
+      connect(speedSensor.w, w)
+        annotation (Line(points={{80,-41},{80,-60},{-80,-60}, {-80,0},{-110,0}}, color={0,0,127}));
+      annotation (
+        Icon(graphics={Text(
+              extent={{-100,-100},{100,-140}},
+              lineColor={28,108,200},
+              textString="%name")}), Documentation(info="<html>
+<p>
+Simple model of a DC PM machine, comprising armature resistance, armature inductance, emf (including torque generation), inertia and pseed measurement.
```
Comment (beutlich) [inline ModelicaTest/Electrical/Machines.mo:489]:
```suggestion
Simple model of a DC PM machine, comprising armature resistance, armature inductance, emf (including torque generation), inertia and speed measurement.
```

**Review by beutlich — COMMENTED** [review]
It is appreciated that you also included the comparisonSignals.txt files.

File: ModelicaTest/Electrical/Machines.mo (hunk 1 above)
Comment (AHaumer) [reply ModelicaTest/Electrical/Machines.mo]: comparisonSignals already included ...

**Review by casella — APPROVED** [review]
LGTM

**Comment by AHaumer** [comment]
@beutlich so why is your approval missing?

**Review by beutlich — COMMENTED** [review]
> 
> 
> @beutlich so why is your approval missing?

I checked and fixed some general issues, but I can't judge on the electrical focus. That should be the job of the library officers.

**Review by christiankral — APPROVED** [review]
I understand these tests redeclared parameter records and parameter propagation applied to DC machine drives. So these examples look good to me.

**Pull request merged**